### PR TITLE
Make runningAsSource a global variable

### PR DESCRIPTION
**Pull request by seanbudd**
### Link to issue number:
Blocking #13254, as this PR needs to check if NVDA is running as source

### Summary of the issue:
NVDA checks in various places if it is running as source.
This check should be abstracted out, as the check is dependent on py2exe behaviour.
When viewing code, it is also not immediately clear what `getattr(sys, 'frozen')` means, whereas a named variable will be clearer.

### Description of user facing changes
A named variable with documentation for code contributors.

### Description of development approach
Checked what `import sys; getattr(sys, 'frozen', None)` returns in the python console of an installed copy of NVDA and a source copy of NVDA.

### Testing strategy:
Smoke test running NVDA from installed and source.

### Known issues with pull request:
None
### Change log entries:

N/A

### Code Review Checklist:

- [x] Pull Request description:
  - description is up to date
  - change log entries
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] API is compatible with existing add-ons.
- [x] Documentation:
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English

### CODE REVIEW

**Review by lukaszgo1 — COMMENTED** [review]
It looks like there are some checks for source VS binary copy in nvda.pyw and nvda_slave.pyw. Have you considered converting these to the new approach while at it?

**Comment by seanbudd** [comment]
Thanks @lukaszgo1 missed these the first time around, fixed now

File: source/globalVars.py [hunk 1]
```diff
@@ -79,3 +80,9 @@ class DefaultAppArgs(argparse.Namespace):
 Note that deprecated code may be imported at runtime,
 and as such, this value cannot be changed at runtime to test compliance.
 """
+
+runningAsSource = getattr(sys, 'frozen', None) is None
```
Comment (feerrenrut) [inline source/globalVars.py:84]:
```python
runningAsSource: bool = getattr...
```

**Review by feerrenrut — APPROVED** [review]

File: source/globalVars.py (hunk 1 above)
Comment (seanbudd) [reply source/globalVars.py:84]:
```suggestion
runningAsSource: bool = getattr(sys, 'frozen', None) is None
```

**Pull request merged**